x-pack/plugins/security_solution/public/common/components/header_page/index.tsx

```diff
@@ -84,27 +84,29 @@ export interface HeaderPageProps extends HeaderProps {
   titleNode?: React.ReactElement;
 }
 
-const HeaderLinkBack: React.FC<{ backOptions: BackOptions }> = React.memo(({ backOptions }) => {
-  const { navigateToUrl } = useKibana().services.application;
-  const { formatUrl } = useFormatUrl(backOptions.pageId);
+export const HeaderLinkBack: React.FC<{ backOptions: BackOptions }> = React.memo(
+  ({ backOptions }) => {
+    const { navigateToUrl } = useKibana().services.application;
+    const { formatUrl } = useFormatUrl(backOptions.pageId);
```

Comment:
anything change in this component? I can't see anything

Reply:
yes, the component is now exported for use below, here https://github.com/elastic/kibana/pull/110429/files#diff-4b2b5818c5d43b7448e0bacf06d9963ce20bb4b9572c32be24655a30dc142146R164